dataactcore/utils/fileF.py

```diff
@@ -191,5 +194,5 @@ def generate_f_rows(submission_id):
                                      submission_grants(submission_id)):
         result = OrderedDict()
         for key, mapper in mappings.items():
-            result[key] = mapper.transform(model_row) or ''
+            result[key] = mapper(model_row) or ''
```

Comment:
I'll add a comment, but a `mapper` is a callable that converts a `model_row` into a CSV cell value. `mappings` is a collection of `mapper`s for each of the CSV columns.

We don't need the `transform` anymore due to changing them `mapper`s to callables (rather than objects with a "transform" method)